cmd/use-context.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	. "github.com/microsoft/go-sqlcmd/cmd/sqlconfig"
+)
+
+func init() {
+	const use = "use-context CONTEXT_NAME"
+	const short = "Set the current-context in a sqlconfig file."
+	const long = short
+	const example = `# Use the context for the sa@sql1 sql instance
+  sqlcmd config use-context sa@sql1`
+
+	var run = func(cmd *cobra.Command, args []string) {
+		var config Sqlconfig
+		var name = args[0]
+
+		viper.Unmarshal(&config)
```

Comment:
viper has its own methods to search config with xpath-like expressions. I am curious whether those methods are faster than this model of using Unmarshal followed by a bunch of loops in the helper methods.
Or are they so complex to use that any perf benefit they haven't isn't worth the developer effort?